src/main/java/com/bieganski/ox/model/Field.java

```diff
@@ -0,0 +1,70 @@
+package com.bieganski.ox.model;
+
+/**
+ * Field represents symbol O or X on given by user position.
+ */
+public class Field implements Comparable<Field> {
+
+  private int position;
+  private Symbol symbol;
+
+  public Field(int position, Symbol symbol) {
+    this.position = position;
+    this.symbol = symbol;
+  }
+
+  /**
+   * Check if field is on given position.
+   * @param position - position to compare
+   * @return true if field is on given position
+   */
+  public boolean isOnPosition(int position) {
+    return this.position == position;
+  }
+
+  /**
+   * Check if field is in given bounds.
```

Comment:
Bounds - plural.
int bounds - plural in name, singular in reality.
The second bound is known only if you read the code. 
This is the issue docs here should have addressed.

Reply:
You are right. Done.